docs/source/fundamentals/benefits.md

```diff
@@ -0,0 +1,133 @@
+---
+title: GraphQL benefits
+description: Why adopting GraphQL and Apollo will help you ship features faster
+---
+
+Managing data for modern applications is difficult. Developers have to aggregate data from multiple sources, distribute it upon multiple platforms, and plumb it into an app's UI. On top of that, front-end developers have to decide how to manage state on the client, all while executing complicated features such as caching and optimistic UI.
```

Comment:
I think we can probably present the problem without calling it "difficult" (See my reasoning/similar concern with the use of "advanced" below).  For example: `Managing data in modern applications can present a number of challenges.`.